applications/source/syslog-source/src/main/resources/META-INF/dataflow-configuration-metadata.properties

```diff
@@ -0,0 +1,2 @@
+configuration-properties.classes=org.springframework.cloud.fn.supplier.syslog.SyslogSupplierProperties
```

Comment:
What is the difference between this file and the previous?
Can't we do the stuff just with one of them?

Reply:
We provide both for SCDF backward compatibility reasons. We are deprecating the file that contains `whitelist`,  but provide here for the time being so that SCDF users can continue to use it on older versions of SCDF.